assets/src/components/propertiesPanel/header.tsx

```diff
@@ -90,10 +119,17 @@ const directionName = (
   route?: Route
 ): string => (route ? route.directionNames[directionId] : "")
 
+const nowInSeconds = (): number => Math.floor(Date.now() / 1000)
+
 const Header = ({ vehicle, route }: Props) => {
   const [{ ladderDirections, settings }, dispatch] = useContext(
     StateDispatchContext
   )
+  const [epocNowInSeconds, setEpocNowInSeconds] = useState(nowInSeconds())
+  useInterval(() => setEpocNowInSeconds(nowInSeconds()), 1000)
```

Comment:
This can use `now()` from `dateTime.ts` instead of defining `nowInSeconds`.

Reply:
Well, we need to compare now with the vehicle timestamp we are getting which is in epoch seconds, so we could start with dates, but we'd have to convert the timestamp instead, and then convert the result into seconds, so this feels a little simpler to me.

Reply:
Ah, right 👍